cypress/support/commands.ts

```diff
@@ -109,3 +109,14 @@ Cypress.Commands.add('requestBase64Image', (url: string) => {
       return Cypress.Blob.blobToBase64String(blob);
     });
 });
+
+Cypress.Commands.add('iFrame', () => {
+  // This creates a lot of entries in the cypress running bar...
+  // in theory adding `{ log: false }` should hide these ...
+  // it does not
+  return cy
```

Comment:
You could reduce the number of log entries by adding `{ log: false }` to the `get` and `its` calls but not `should` [(yet?)](https://github.com/cypress-io/cypress/issues/7693) . Idk if that's really any clearer though; you just lose context for the `should`

Reply:
i think that's a reasonable change, given how these obfuscate all the other content